docs/samples/explanation/alibi/alibiexplainer/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
you need to update your poetry version to 1.8.3 at least.

Reply:
interesting. I removed the poetry and generated it. Why it used old version ??

Reply:
it is because of local installed poetry version. I updated it and regenerated it.